main_test.go

```diff
@@ -88,13 +87,23 @@ func TestVersionArg(t *testing.T) {
 	}
 }
 
+// test cases:
+//
+// -state and -statefile together should error
+//
+// these should all be ok and functionally equivalent:
+//   terraputs -state "$(cat stateFile)"
+//   terraputs -statefile stateFile
```

Comment:
I'm a bit confused about line 96. There is no `-statefile` flag, no?

Reply:
ah my bad. good eye. i listed out all of the expected test cases before shuffling test tables around. i'd originally intended to add in the `statefile <path>` option as well and check that both `-state` and `-statefile` weren't provided but didn't end up adding `-statefile`. we can add that argument in now or in a follow-up PR. i'll change that comment to reflect whatever we decide on.